src/libAtomVM/term.h

```diff
@@ -148,6 +148,8 @@ extern "C" {
 #define BOXED_FUN_SIZE 3
 #define FLOAT_SIZE (sizeof(float_term_t) / sizeof(term) + 1)
 #define REF_SIZE ((int) ((sizeof(uint64_t) / sizeof(term)) + 1))
+#define TERM_BOXED_PID_REF_SIZE (REF_SIZE + 1)
```

Comment:
For consistency with the work we are doing lately I suggest renaming all `PID_REF` and `pid_reference` to `PROCESS_REF` and `process_reference`